app/code/Magento/BraintreeGraphQl/README.md

```diff
@@ -1,4 +1,9 @@
-# BraintreeGraphQl
+# Magento_BraintreeGraphQl module
 
-**BraintreeGraphQl** provides type and resolver for method additional
-information.
\ No newline at end of file
+The Magento_BraintreeGraphQl module provides type and resolver for method additional information. 
```

Comment:
This sentence seems incomplete. Does the module provide the type and resolver methods? What are we referring to when we say "additional information"?

Reply:
Thanks @jeff-matthews I will review